### PR TITLE
Chore/added examples

### PR DESCRIPTION
**Pull request by brenkao**

### CODE REVIEW

File: examples/basic-example/example.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"""A basic prompt for recommending books on {topic}"""
```
Comment (willbakst) [inline examples/basic-example/example.py:1]: update {topic} to "a topic"

File: examples/basic-example/example.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    """
+    Can you recommend some books on {topic}?
+    """
+
+    topic: str
+
+
+prompt = BookRecommendationPrompt(topic="how to bake a cake")
+model = MirascopeChatOpenAI(api_key=os.getenv("OPENAI_API_KEY"))
+res = model.create(prompt)
```
Comment (willbakst) [inline examples/basic-example/example.py:17]: add `str(res)` here as the final line

File: examples/basic-example/requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+mirascope==0.1.0
```
Comment (willbakst) [inline examples/basic-example/requirements.txt:1]: this also requires openai

File: examples/chain-example/example.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+"""A multi-chain prompt for recommending books on {topic} and then asking which one is the best for beginners."""
```
Comment (willbakst) [inline examples/chain-example/example.py:1]: change {topic} to "a topic"

File: mirascope/cli/cli.py [hunk 5]
```diff
@@ -54,7 +54,8 @@ def main():
 
     # Adding 'init' command
     parser_init = subparsers.add_parser("init", help="Initialize mirascope project")
-    parser_init.add_argument("directory_name", help="Main mirascope directory")
+    parser_init.add_argument("--mirascope_location", help="Main mirascope directory")
```
Comment (willbakst) [inline mirascope/cli/cli.py:57]: please re-merge with main after last merge so this doesn't show up?

File: examples/chain-example/requirements.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+mirascope==0.1.0
```
Comment (willbakst) [inline examples/chain-example/requirements.txt:1]: also requires openai

File: examples/basic-example/example.py (hunk 2 above)
Comment (brenkao) [reply examples/basic-example/example.py:17]: you mean a print statement printing str(res)?

File: examples/basic-example/requirements.txt (hunk 3 above)
Comment (brenkao) [reply examples/basic-example/requirements.txt:1]: does installing mirascope not install openai?

File: examples/basic-example/example.py (hunk 2 above)
Comment (willbakst) [reply examples/basic-example/example.py:17]: yeah

File: examples/chain-example/example.py [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    model = OpenAIChat(api_key=os.getenv("OPENAI_API_KEY"))
+    book_recommendation_prompt = BookRecommendationPrompt(topic=topic)
+    book_list = model.create(book_recommendation_prompt)
+
+    best_for_beginners_prompt = BestForBeginnersPrompt(book_list=str(book_list))
+    best_for_beginners = model.create(best_for_beginners_prompt)
+    return str(best_for_beginners)
+
+
+generate_best_book_for_beginners("how to bake a cake")
```
Comment (willbakst) [inline examples/chain-example/example.py:36]: wrap this in a print?

File: examples/basic-example/requirements.txt [hunk 8]
```diff
@@ -0,0 +1,2 @@
...
+openai==1.6.0
```
Comment (willbakst) [inline examples/basic-example/requirements.txt:2]: I realized later that yeah mirascope will install this so not necessary unless we want to make it clear you're installing openai?

**Review by willbakst — APPROVED** [review]

File: examples/basic-example/requirements.txt (hunk 8 above)
Comment (brenkao) [reply examples/basic-example/requirements.txt:2]: i think this is fine to keep, we will update this as necessary with different models

**Pull request merged**